R/databricks.R

```diff
@@ -14,7 +14,7 @@
 #' [standard environment variables](https://docs.databricks.com/en/dev-tools/auth.html#environment-variables-and-fields-for-client-unified-authentication).
 #'
 #' @inheritParams DBI::dbConnect
-#' @param HTTPPath To query a cluster, use the HTTP Path value found under
+#' @param httpPath To query a cluster, use the HTTP Path value found under
```

Comment:
Yes, odbc itself is case-insensitive, and I think we should try to move consistently to once case style in all our documentation and examples. I've filed #641 to ensure that we give better diagnostics when confronted with multiple arguments that differ only in case.